starter-repos/shared-ci-cd/appctl/prepare.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+prepare-manifests:
+  only:
+    refs:
+    - master
+  stage: prepare-manifests
+  image: gcr.io/cloud-solutions-images/appctl
+  script:
+  - |
+    cat > /root/.appctlconfig <<EOF
+    personal_access_tokens:
+    - host: ${CI_SERVER_HOST}
+      token: ${GIT_ACCESS_TOKEN}
+    EOF
```

Comment:
(I'm not sure about this)
From Haiyan's change on creating ssh pair. Should you refer to the newly generated pair rather than the ones from /root/.ssh/id_rsa?

Reply:
It is using Haiyan's generated ssh key pair. A variable named `MANIFEST_WRITER_KEY` will be added with Haiyan's change and we are using the value in `/root/.ssh/id_rsa`.

Reply:
ah I see.